AppGenGuide.md

```diff
@@ -258,32 +258,34 @@ This means that, in order to be able to make additional configuration, one must:
 
 ### FluxCD Manifest
 
-[//]: # (TODO: Validate this information - what needs validation?)
+> _NOTE_: this section needs improvement.
 
 Most of the generated manifest won't need any modification, but for every 
-helm-chart in the `app_manifest.yaml`, a static overrides file will be created.
-The static overrides contain all information that is not to be overwritten
-inside the values.yaml of the helm-chart.
+Helm chart in the `app_manifest.yaml`, a static-overrides file will be created.
+The static-overrides file contains all information that is not to be
+overwritten inside the values.yaml of the Helm chart.
 
 ### Plugins
 
-The generate will create 3 main plugins, the helm, the kustomize and the lifecycle.
+The StarlingX App Generator will create 3 main plugins: the Helm,
```

Comment:
why would helm be capitalize and the other won't?

Reply:
Because Helm is the name of a tool. Maybe `kustomize` should be between backticks.